app/src/main/res/values/strings.xml

```diff
@@ -429,4 +429,5 @@
   <string name="coming_soon">Coming Soon</string>
   <string name="recommended_stories">Recommended Stories</string>
   <string name="stories_for_you">Stories For You</string>
+  <string name="recently_played_activity_label">Recently played stories</string>
```

Comment:
Will we be using ``title`` instead of ``label`` per the discussion from the team meeting this week?